src/ui/components/LedgerAdmin.js

```diff
@@ -74,38 +87,60 @@ export const LedgerAdmin = (): ReactNode => {
     let nextLedger;
     if (ledger.account(id).active) {
       nextLedger = ledger.deactivate(id);
-      setCheckBoxSelected(false);
     } else {
       nextLedger = ledger.activate(id);
-      setCheckBoxSelected(true);
     }
     updateLedger(nextLedger);
   };
 
   const resetIdentity = () => {
     setIdentityName("");
     setSelectedId(null);
-    setCheckBoxSelected(false);
     setPromptString("Add Identity: ");
   };
 
   const setActiveIdentity = (identity: Identity) => {
     setIdentityName(identity.name);
     setSelectedId(identity.id);
-    setCheckBoxSelected(ledger.account(identity.id).active);
     setPromptString("Update Identity: ");
   };
 
+  const loadEntitiesCheckList = () => {
+    let identities = {};
+
+    ledger
+      .accounts()
+      .map((a) => a.identity)
+      .map((identity) => {
+        identities = {
+          ...identities,
+          [`${identity.id}`]: ledger.account(identity.id).active,
+        };
+      });
```

Comment:
Hey @crisog, I'd like to suggest a simplified version of this code as follows:
```suggestion
    const identities = {};

    ledger.accounts().map(({identity, active}) => {
      identities[`${identity.id}`] = active;
    });
```

It eliminates the second `map()`, prevents re-spreading the `identities` object each time, and eliminates the need to access `ledger.account(id)` to get each active status. I've pulled your branch and tested this locally, and it seems to work fine though I'm new to the project and I'd suggest testing for yourself in case I missed something.